src/vs/workbench/browser/parts/titlebar/media/titlebarpart.css

```diff
@@ -67,83 +92,97 @@
 	margin-left: auto;
 }
 
-.monaco-workbench .part.titlebar > .titlebar-container > .window-title {
+/* Window title text */
+.monaco-workbench .part.titlebar>.titlebar-container>.window-title {
 	flex: 0 1 auto;
 	font-size: 12px;
 	overflow: hidden;
 	white-space: nowrap;
 	text-overflow: ellipsis;
 	margin-left: auto;
 	margin-right: auto;
-	zoom: 1; /* prevent zooming */
-}
-
-.monaco-workbench .part.titlebar > .titlebar-container.enable-title-menu > .window-title {
-	display: none;
 }
 
 /* Windows/Linux: Rules for custom title (icon, window controls)  */
-.monaco-workbench.web .part.titlebar > .titlebar-container,
-.monaco-workbench.windows .part.titlebar > .titlebar-container,
-.monaco-workbench.linux .part.titlebar > .titlebar-container {
+.monaco-workbench.web .part.titlebar>.titlebar-container,
```

Comment:
Any reason you changed the style here? We typically have whitespace around `>`

Reply:
some formatter I have has been doing this on occasion. and I just recently found that I had the setting which only applies this to changed regions

Reply:
Can we revert that as part of `sbatten/outdoor-mosquito` maybe? I prefer the old style.